### PR TITLE
fix baggage not work in trace demo app.

### PR DESCRIPTION
**Pull request by JaredTan95**
Signed-off-by: jian.tan <jian.tan@daocloud.io>

**Description:** fix baggage not work in trace demo app.

refs: https://github.com/open-telemetry/opentelemetry-go-contrib/blob/main/instrumentation/net/http/otelhttp/example/client/client.go#L52

### CODE REVIEW

**Comment by JaredTan95** [comment]
`Skip Changelog`

**Comment by JaredTan95** [comment]
Skip Changelog

**Review by jpkrohling — APPROVED** [review]

**Comment by JaredTan95** [comment]
> I think a changelog entry would indeed be useful for this.

Sorry, I don't fully understand what's your meaning. Should I update something?

**Comment by jpkrohling** [comment]
> Should I update something?

Yes, the changelog file at the root of this repository, to inform users about this change.

**Comment by JaredTan95** [comment]
> 

Ok, I didn't think it was a very important modification before, so I skipped it with `Skip Changelog`.
I will update it.

File: examples/demo/server/main.go [hunk 1]
```diff
@@ -156,6 +158,11 @@ func main() {
 		requestCount.Add(ctx, 1, commonLabels...)
 		span := trace.SpanFromContext(ctx)
 		span.SetAttributes(serverAttribute)
+		bag := baggage.FromContext(ctx)
+		for i, member := range bag.Members() {
+			span.AddEvent(fmt.Sprint("baggage member: ", i), trace.WithAttributes(attribute.String(member.Key(), member.Value())))
```
Comment (cijothomas) [inline examples/demo/server/main.go:163]: is this code reading baggage, and adding them as spanevents? Why not simply add them as span attributes?

Comment (JaredTan95) [reply examples/demo/server/main.go:163]: make sense, I will update it.

**Review by codeboten — COMMENTED** [review]
Please rebase and fix lint 👍

**Comment by JaredTan95** [comment]
> Please rebase and fix lint 👍

done.

**Pull request merged**